.travis/runcoveralls.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+# SPDX-License-Identifier: MIT
+
+# Reports coverage results using coveralls. The aim of this script is to
+# provide a unified way to reporting coverage results across all linux system
+# roles projects.
+
+# The given command line arguments are passed to coveralls.
+
+# Environment variables:
+#
+#   LSR_PUBLISH_COVERAGE
```

Comment:
It seems that the change to set LSR_PUBLISH_COVERAGE to strict is missing. Should it be added to config.sh?

Reply:
What change would that be?  The initial commit of runcoveralls.sh to the template is here - https://github.com/linux-system-roles/template/commit/57513dac636bb8fef4e9fcc3c95066f6b704ed16#diff-9c9e55de5f8cf641950dc28851dbd328 - I don't see where the default value of `LSR_PUBLISH_COVERAGE` would have been `strict`

Reply:
I believe the default value in the template should be `strict`.

Reply:
done

Reply:
The default value of `LSR_PUBLISH_COVERAGE` should be left empty, since afaik at now only network role is using coverage & coveralls. Setting the `LSR_PUBLISH_COVERAGE` is done in `.travis/config.sh`, so to enable coverage reporting just put `LSR_PUBLISH_COVERAGE=strict` or `LSR_PUBLISH_COVERAGE=yes` there. Just a note, I am not sure what are @tyll expectations on coverage reporting, there is a discussion in #120 about it. Similar PR with .travis/runcoveralls.sh is also #152.

Reply:
Ok, I see.  The coveralls stuff is still WIP.  I suggest then that this PR is ok, and most of the work being done for https://github.com/linux-system-roles/network/pull/120 and https://github.com/linux-system-roles/network/pull/152 should really go into template (except the .travis/config.sh changes), then sync'd back to network (and the other roles).

Reply:
> The default value of `LSR_PUBLISH_COVERAGE` should be left empty, since afaik at now only network role is using coverage & coveralls. Setting the `LSR_PUBLISH_COVERAGE` is done in `.travis/config.sh`, so to enable coverage reporting just put `LSR_PUBLISH_COVERAGE=strict` or `LSR_PUBLISH_COVERAGE=yes` there. Just a note, I am not sure what are @tyll expectations on coverage reporting, there is a discussion in #120 about it. Similar PR with `.travis/runcoveralls.sh` is also #152.

I guess the value for the network role should be `yes` then, because it should be enabled and (even if it is broken) and not accidentally be disabled IMHO. Please note, the value `yes` is not mentioned in the documentation for the variable, it only mentions emtpy, `strict` and `debug`. This is why I proposed `strict`.